### PR TITLE
Bump FluentD Chart

### PR DESCRIPTION
**Pull request by djsd123**
Bumped chart version

Added CHANGELOG

### CODE REVIEW

File: charts/fluentd/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [0.2.0] - 2018-04-25
+### FluentD Tuning
+- Modifying to currently working config set.
+  Particularly ensuring we are capturing systemd events from journalD as are current distro is
```
Comment (andyhd) [inline charts/fluentd/CHANGELOG.md]: as *our* current distro is debian jessie

File: charts/fluentd/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  debian jessie
+  
+- Removing log sources that do not exist on systemd distros i.e.
+  `/var/log/kubelet`.  Mentioned path will only exists on sysV init
+  distros.
+
+- Moving to supported K8s addon release of fluentD-elasticsearch
+
+- Increasing slow_flush_log_threashhold to prevent the corresponding
```
Comment (andyhd) [inline charts/fluentd/CHANGELOG.md]: `s/threashold/threshold/`

Comment (djsd123) [reply charts/fluentd/CHANGELOG.md]: Ha, ha, thanks

File: charts/fluentd/CHANGELOG.md (hunk 1 above)
Comment (djsd123) [reply charts/fluentd/CHANGELOG.md]: +1 for your grammar skills

**Pull request merged**